.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # These owners will be the default owners for everything in the repo and
 # will be requested for review when someone opens a pull request.
-*       @cake-contrib/team-bbt @jokay
\ No newline at end of file
+*       @cake-contrib/team-bbt @pascalberger @jokay
```

Comment:
please add @bbtsoftware/dev-services and remove @cake-contrib/team-bbt and @jokay

Reply:
Done with #147